docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -137,15 +137,22 @@ PacemakerResourcesToStop=("openstack-cinder-volume"
 
 echo "Stopping systemd OpenStack services"
 for service in ${ServicesToStop[*]}; do
-    for i in {1..3}; do
-        SSH_CMD=CONTROLLER${i}_SSH
-        if [ ! -z "${!SSH_CMD}" ]; then
-            echo "Stopping the $service in controller $i"
-            if ${!SSH_CMD} sudo systemctl is-active $service; then
-                ${!SSH_CMD} sudo systemctl stop $service
-            fi
+ifeval::["{build_variant}" != "ospdo"]
+for i in {1..3}; do
```

Comment:
This still breaks indentation of the code example. See the indentation on what you're removing vs on what you're adding.

Reply:
ok